modules/core/src/test/java/org/apache/ignite/internal/binary/BinaryFieldExtractionSelfTest.java

```diff
@@ -193,6 +193,47 @@ public void testChangeTypeIdOfBinaryFieldCaseNotFoundExpectedTypeId() throws Exc
         assertThrows(log, () -> timeBinField.value(timeValBinObj), BinaryObjectException.class, expMsg);
     }
 
+    /**
+     * Checking the exception and its text when changing the typeId by spoil
```

Comment:
why a u using "the" here ? )

Reply:
what the main purpose of such test ? to check correct err message ?

Reply:
Check that the correct exception and text.